ambry-api/src/main/java/com/github/ambry/account/ContainerBuilder.java

```diff
@@ -116,6 +119,14 @@ public ContainerBuilder setStatus(ContainerStatus status) {
     return this;
   }
 
+  /**
+   * Sets the last update time of the {@link Container} to build.
+   */
+  public Long setLastUpdateTime(Long lastUpdateTime) {
```

Comment:
this is a bad signature in builder. It should be 
```
public ContainerBuilder lastUpdateTime(long val) {
    this.lastUpdateTime = val;
    return this;
}
```
It's up to you to prefix `set` to the method name. I would not, but to keep it consistent with all the methods in this file, we probably should.

Reply:
Since we have already started using set for the other methods, and this builder is shared with Nuage, I think we should keep prefixing these with `set`. Not sure how this builder ended up having different naming conventions than all the other ones though.

Reply:
Return ContainerBuilder instead of Long type and keep name prefixing with set for consistency.